tests/hazmat/backends/test_openssl.py

```diff
@@ -555,18 +555,9 @@ def test_numeric_string_x509_name_entry(self):
             x509.load_der_x509_certificate,
             backend
         )
-        if backend._lib.CRYPTOGRAPHY_OPENSSL_LESS_THAN_102I:
```

Comment:
Don't we still support < 1.0.2i? Xenial ships with 1.0.2g so we should still need this branch